src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -897,7 +901,7 @@ export function DistributionForm({
           ) : vaults[0] ? (
             <Button
               color="secondary"
-              disabled={giftSubmitting || !sufficientGiftTokens}
+              disabled={giftSubmitting || !sufficientGiftTokens || !contracts}
```

Comment:
Ithink this is breaking the tests because it adds another invariant where contracts being defined makes this button NOT disabled, where it should be disabled.

Reply:
actually, nvm, that doesn't make sense.